staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -7314,6 +7314,7 @@ type LimitRangeItem struct {
 // LimitRangeSpec defines a min/max usage limit for resources that match on kind.
 type LimitRangeSpec struct {
 	// Limits is the list of LimitRangeItem objects that are enforced.
+	// +optional
```

Comment:
this will probably require updating generated files by running `make update` and adding the resulting changes to the PR